libs/deserializer/src/main/java/de/cyface/deserializer/v1/BinaryFormatParser.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (C) 2020 Cyface GmbH - All Rights Reserved
+ *
+ * This file is part of the Cyface Server Backend.
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ */
+package de.cyface.deserializer.v1;
```

Comment:
see above, also a copy of an existing class (from before we ported to the new binary format)